test/src/api/commands/element/testWaitForElementNotPresent.js

```diff
@@ -73,4 +79,40 @@ describe('waitForElementNotPresent', function () {
       strictEqual(commandInstance.rescheduleInterval, 10);
     });
   });
+
+
+  it('client.waitForElementNotPresent() report should not contain error in case of success', async function() {
+    MockServer.addMock({
+      url: '/wd/hub/session/1352110219202/elements',
+      method: 'POST',
+      postdata: {
+        using: 'css selector',
+        value: '#badElement'
+      },
+      response: JSON.stringify({
+        value: []
+      })
+    }, true, true);
+
+    const testsPath = [
+      path.join(__dirname, '../../../../sampletests/withwait/elementNotPresent.js')
+    ];
+
+    const globals = {
+      calls: 0,
+      reporter(results) {
+        assert.strictEqual(globals.calls, 2);
+        assert.strictEqual(Object.keys(results.modules).length, 1);
+        assert.ok('elementNotPresent' in results.modules);
+        assert.strictEqual(results.modulesWithEnv.default.elementNotPresent.completedSections.demoTest.commands[1].status, 'pass');
+        assert.strictEqual(results.modulesWithEnv.default.elementNotPresent.completedSections.demoTest.commands[2].status, 'fail');
+      }
+    };
+
+    await NightwatchClient.runTests(testsPath, settings({
+      globals
+    }));
+
+    return true;
```

Comment:
From @beatfactor, why are we returning true?

Reply:
not required, updated.